text/0011-namespaces.md

```diff
@@ -0,0 +1,29 @@
+# Namespaces 
+- Start Date: 2019/06/17
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/11
+
+## Base Namespace 
+The base namespace for maps and modules will be `@openmrs`
+
+## Core Features 
+The namespace for core features will be `@openmrs/core` which will include core atoms 
+
+## Module based namespaces 
+The namespace for module based features, which will leverage existing modules will be `@openmrs/module/modulename` for example 
+- `@openmr/module/appointmentscheduling` for appointment scheduling 
+
+## Implementing Partner and Distributions 
+The namespace for partners will be as follows
+- `@openmrs/reference-application/` for the Reference Application 
+- `@openmrs/ugandaemr/` for UgandaEMR 
+- `@openmrs/ampath/` for Ampath specific needs 
+- `@openmrs/hiv` for an HIV based distribution 
```

Comment:
This last example seems rife for conflicts. The top level path should identify the "authority" responsible for anything within that path & below:
* `module` – module authors are responsible for using a unique module name
* `reference-application` – the team working on the OpenMRS Reference Application
* `ampath`, `pih`, etc. – the organization (ideally would align with owned domain name)
* `bahmni`, `ugandaemr`, etc. – the distribution (ideally would align with owned domain name)
* `hiv` – **who owns this namespace?**

Reply:
@bmamlin Thanks for the review, thinking now of a distro prefix for distributions and an organization at the root - this provides segregation at that level